include/vkgcDefs.h

```diff
@@ -378,6 +379,14 @@ enum class ThreadGroupSwizzleMode : unsigned {
   Count
 };
 
+/// Represents mapping layout of the resources used in shaders
+enum class ResourceMappingLayout : unsigned {
+  // Compact scheme make full use of all the user data registers.
+  Compact = 0,
+  // The searching path of resource belongs to a specific binding is fixed in indirect scheme.
```

Comment:
nit: The search path?

Reply:
This comment is copied from driver, but it is really hard to understand.